### DISCUSSION TITLE
Using Silk.NET in Eto.Forms - possible?

### DISCUSSION THREAD

**Discussion by philstopford** [dotnet/Silk.NET#1585] [Q&A]
I have a set of tools written in Eto.Forms (to take advantage of its common UI code mapping to native interfaces via WPF and GTK# on Windows and Linux). There is a Veldrid component for Eto.Forms, but it has issues for some configurations.

I was curious whether it might be possible to use Silk.NET with Eto.Forms as a result.

**Comment by Beyley** [comment]
> I have a set of tools written in Eto.Forms (to take advantage of its common UI code mapping to native interfaces via WPF and GTK# on Windows and Linux). There is a Veldrid component for Eto.Forms, but it has issues for some configurations.
> 
> I was curious whether it might be possible to use Silk.NET with Eto.Forms as a result.

possible, but no code has been written tmk
